repository-snapshot/Dockerfile

```diff
@@ -0,0 +1,35 @@
+# Use the specified base image
+FROM ubuntu:24.04
+
+# Installing dependencies
+RUN apt-get update && apt-get install -y \
+  coreutils           \
+  curl                \
+  createrepo-c        \
+  cron                \
+  debianutils         \
+  wget                \
+  rsync
+
+# Adding Enableit repo and key
+RUN wget https://repos.obmondo.com/packagesign/public/apt/pubkey.gpg -O /etc/apt/keyrings/obmondo_custom.asc
+RUN echo "deb [arch=amd64 signed-by=/etc/apt/keyrings/obmondo_custom.asc] https://repos.obmondo.com/packagesign/public/apt noble main" | tee -a /etc/apt/sources.list.d/obmondo_custom.list
+
+# Installing obmondo-scripts-common and obmondo-repository-mirror
+RUN apt-get update && apt-get install -y \
+  obmondo-scripts-common \
+  obmondo-repository-mirror
+
+# Make sure the snapshot_repo script is executable
+RUN chmod +x /usr/local/bin/snapshot_repo
```

Comment:
where is this script coming

Reply:
This script is coming from   obmondo-repository-mirror packages which already have the exec permission 
have removed the permission line